src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -66,19 +68,15 @@ internal void LogDiagnostic(Diagnostic diagnostic, CultureInfo culture)
                 message = "<None>";
             }
 
-            string description = diagnostic.Descriptor.Description.ToString(culture);
-            if (string.IsNullOrEmpty(description))
-            {
-                _writer.Write("fullMessage", message);
-            }
-            else
+            _writer.Write("message", message);
+
+            if (diagnostic.IsSuppressed)
             {
-                _writer.Write("shortMessage", message);
-                _writer.Write("fullMessage", description);
+                _writer.WriteArrayStart("suppressionStates");
+                _writer.Write("suppressedInSource");
+                _writer.WriteArrayEnd();
             }
 
-            _writer.Write("isSuppressedInSource", diagnostic.IsSuppressed);
-
             WriteTags(diagnostic);
```

Comment:
Yes, sorry, that was the last (hah) breaking change. I merged it this morning.

---
In reply to: [62950352](https://github.com/dotnet/roslyn/pull/11233#discussion_r62950352) [](ancestors = 62950352)